libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryClient.java

```diff
@@ -35,6 +34,10 @@ public TelemetryClient(String accessToken, String userAgent, TelemetryClientSett
     this.logger = logger;
   }
 
+  public void setUserAgent(String userAgent) {
```

Comment:
Why does this need to be `public`?

Reply:
get/set nomenclature is hijacked by the Java language and should be only used when defining POJOs/DTOs.